src/distributions/markovchain.jl

```diff
@@ -0,0 +1,88 @@
+using Setfield
+
+export MarkovChain
+
+"""
```

Comment:
We should add something here to emphasize that this is experimental. I think it has a lot of potential, but may need some smoothing out.

Reply:
I've added a note here:
https://github.com/cscherrer/Soss.jl/pull/137/commits/4e2ecb920a5894cdb3f4b6cff46fccb51c680776